src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -93,114 +87,112 @@ public ServiceBusHelper serviceBusHelper() {
     }
 
     @Scheduled(fixedDelayString = "${scheduling.task.scan.delay}")
-    public void processBlobs() throws IOException, StorageException, URISyntaxException {
-        for (CloudBlobContainer container : cloudBlobClient.listContainers()) {
+    public void processBlobs() throws IOException {
+        List<ContainerItem> containerItems = azureStorageHelper.listContainers().blockingGet().body().containerItems()
+            .stream()
+            .filter(c -> !c.name().equals("$web"))
+            .collect(toList());
+
+        log.info("Processing blobs in containers: {}", containerItems.stream()
```

Comment:
Umm ... this should be a debug log and surrounded by isDebug statement otherwise we are getting the list of items every time. Waste of time and churn in memory.